### ISSUE TITLE
Fonts a bit too small for easy reading

### ISSUE THREAD

**Issue by matta** [vilterp/react-blog#7]
Hello there. I came across your blog by way of Hacker News. I can't claim to understand what "Turning the IDE Inside Out with Datalog" even means, but I've long wanted a better grasp of logic programming in general, so I intend to give it a good read later.

I thought I'd mention that your blog uses fonts that are too small for me to read comfortably, in case you are interested in an opportunity to make your site more accessible.

I've adjusted my browser's default font size to 20px, which makes most text easily readable, but your blog's CSS overrides this user setting with a fixed font size in `px`. This is a common practice, though an outdated one as today's web standards make it pretty easy to base font sizes off the user's default.

The TLDR is that Github CSS, which your blog uses, hard codes body text to `16px` which overrides my default browser font size.  Hardly anybody notices because everyone with normal descent eyesight finds `16px` readable -- it is the "default default" browser font size.  Sadly, it looks like Github CSS hard codes `px` font sizes pervasively so...no I guess there is no easy answer if you want to keep using it.

See https://www.rfc20.org/posts/please-respect-my-browser-font-size/ for more info.
